modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-cluster-wide-migration_{context}"]
+= About cluster-wide migration
+
+During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other one is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps in order to avoid possible conflicts between the two control planes. The following topics describe the required conditions for successful migration.
```

Comment:
```suggestion
During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps to avoid possible conflicts between the two control planes.
```

Reply:
Also, this has `{SMv2Version}`, which renders to 2.6.6 currently, but the content below all has hardcoded `2.6`. Which one is better to use?

Reply:
The migration from service mesh 2.6 to 3.0 is very complicated. These are two different products - not merely a software bump. Gwynne and I talked through the migration workflows (Gwynne is working on multitenant and I'm working on cluster-wide) we thought it would be more effective to use 2.6 when referring to the general installation and use the variable when the actual version is more relevant to the specific context.

Reply:
Ok, good deal - if the difference is intended and serving a purpose then SGTM :)